### DISCUSSION TITLE
Exists some documentation of the project

### DISCUSSION THREAD

**Discussion by eduvhc** [gritcsenko/HomeInventory#770] [General]
I follow this project very close, there's some documentation to explain the domain.

Btw I really appreciate your patterns and engineering!

**Comment by gritcsenko** [comment]
Hey @hashproton! Thanks for reaching out!
I've created PR #918 to work on it.
TBH I do not have clear picture yet and experimenting with architecture to figure out what will work best
Anyway, I promise to document as much as possible.
